### PR TITLE
Add NHC 0.7 configs and a script for creating those

### PR DESCRIPTION
**Pull request by slintes**

### CODE REVIEW

**Comment by slintes** [comment]
/pj-rehearse pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-test
/pj-rehearse pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-images
/pj-rehearse pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-ci-bundle-my-bundle
/pj-rehearse pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-openshift-e2e

**Comment by slintes** [comment]
/pj-rehearse pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-test pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-images pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-ci-bundle-my-bundle pull-ci-medik8s-node-healthcheck-operator-release-0.7-4.14-openshift-e2e

File: ci-operator/config/medik8s/common/new-release.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# copy all main config files
+for file in $(ls | grep main); do
+  # rename the file with branch name
+  new_file=$(echo ${file} | sed "s/main/${BRANCH}/")
+  cp ${file} ${new_file}
+  # update the branch name in the config
+  sed -i "s/branch: main/branch: ${BRANCH}/g" ${new_file}
+done
+
+echo "Done, please run 'make update' for creating jobs"
```
Comment (clobrano) [inline ci-operator/config/medik8s/common/new-release.sh:38]: why not execute `make update` directly in this script?

Comment (slintes) [reply ci-operator/config/medik8s/common/new-release.sh:38]: I wanted to have the chance to review the changes the script is doing

File: ci-operator/config/medik8s/common/new-release.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+cd ${REPO}
+
+# copy all main config files
+for file in $(ls | grep main); do
+  # rename the file with branch name
+  new_file=$(echo ${file} | sed "s/main/${BRANCH}/")
+  cp ${file} ${new_file}
+  # update the branch name in the config
+  sed -i "s/branch: main/branch: ${BRANCH}/g" ${new_file}
```
Comment (clobrano) [inline ci-operator/config/medik8s/common/new-release.sh:35]: have you considered using https://github.com/mikefarah/yq ?

```
yq --inplace '.zz_generated_metadata.branch = "${BRANCH}"' ${new_file}
```

Comment (slintes) [reply ci-operator/config/medik8s/common/new-release.sh:35]: that's nicer for sure, thanks. But does everyone have yq installed? 🤔

Comment (clobrano) [reply ci-operator/config/medik8s/common/new-release.sh:35]: It's surely less ubiquitous than sed :)

**Pull request merged**